packages/app/src/background/services/bandada/BandadaService.ts

```diff
@@ -58,9 +58,16 @@ export class BandadaService {
     const response = await fetch(`${API_URL}/groups/${groupId}/members/${hexToBigint(identity.commitment)}`, {
       method: "GET",
       headers: DEFAULT_HEADERS,
-    }).then((res) => res.json() as Promise<string>);
+    });
+
+    if (!response.ok) {
+      const result = (await response.json()) as { message: string };
+      throw new Error(result.message.toString());
+    }
+
+    const result = (await response.json()) as string;
```

Comment:
```suggestion
    const result = (await response.json()) as unknown;

    if (!response.ok) {
      throw new Error((result as { message: string }).message);
    }

    return JSON.parse(result as string) as boolean;
```

Reply:
I'll fix this in this PR https://github.com/CryptKeeperZK/crypt-keeper-extension/pull/873